package.json

```diff
@@ -36,7 +36,7 @@
     "node-libs-react-native": "^1.2.1",
     "qrcode": "^1.4.1",
     "react": "18.0.0",
-    "react-dom": "^16.4.2",
+    "react-dom": "18.0.0",
```

Comment:
@siddarthkay this pr updates react-dom to match react version, these should always be in sync 👌

Reply:
Ooh my bad, Thanks @J-Son89 !